src/components/dictionaryConcepts/containers/CreateConcept.jsx

```diff
@@ -45,7 +44,7 @@ export class CreateConcept extends Component {
       descriptions: PropTypes.array,
     }).isRequired,
     addedConcept: PropTypes.array.isRequired,
-    addNewAnswer: PropTypes.func.isRequired,
+    // addNewAnswer: PropTypes.func.isRequired,
```

Comment:
@AGMETEOR , kindly remove the commented code.

Reply:
@Duncanian I have removed it.